Content/Items/Palestone/Weapons.PalestoneNail.cs

```diff
@@ -187,83 +213,69 @@ public override void AI()
 				}
 				else
 				{
-					Projectile.ai[1] = -1;
+					EnemyWhoAmI = -1;
 					foundTarget = false;
 				}
 			}
 
-			if (!foundTarget)
+			if (!Player.HasMinionAttackTargetNPC)
 			{
-				if (Projectile.ai[1] < -enemyCheckDelay)
```

Comment:
This removes the delay of 30 ticks between searching for a new target. _It used EnemyWhoAmI in the negative range as the timer so that it was auto synced._
Either this should be reimplemented, or the const **enemyCheckDelay** should be removed.